civet/compilers/es6.py

```diff
@@ -0,0 +1,57 @@
+import errno
+import os
+
+from civet.compilers.base_compiler import Compiler
+
+
+def mkdir_p(path):
+    try:
+        os.makedirs(path)
+    except OSError as exc:  # Python >2.5
```

Comment:
Although I see that you got this snippet of code from http://stackoverflow.com/questions/600268/mkdir-p-functionality-in-python, can we remove the `Python >2.5` comment? I don't think we need to support any version of Python below 2.7.